### PR TITLE
Set up main site pages and navigation

### PR DESCRIPTION
**Pull request by xujustinj**
Uses React Router to handle navigation and bootstrap to do some basic formatting

Also got a bit ahead of myself and implemented the forms for querying (searching/adding) businesses

We prototyped our design in https://www.figma.com/file/TWfebgvSWsTjmgEAIvhcSz/Web-Client?node-id=0%3A1 and then completely ignored it

### CODE REVIEW

File: src/components/Header.tsx [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+	align-items: center;
+	justify-content: ${({ align }): JustifyContentProps => align || "inherit"};
+
+	height: ${dimensions.headerHeight};
+	width: ${({ weight }) => `calc(${weight ?? 1} * 100%)`};
+`;
+
+const HeaderLogo = styled(Link)`
+	color: ${colors.logoText};
+	font-size: 32px;
```
Comment (ScottMorris) [inline src/components/Header.tsx]: Does React support relative font sizes?

File: src/styles/dimensions.ts [hunk 2]
```diff
@@ -0,0 +1,9 @@
+export const dimensions = Object.freeze({
+	headerHeight: "64px",
```
Comment (ScottMorris) [inline src/styles/dimensions.ts]: Seeing raw pixel sizes is kinda gross, but perhaps the HTML world still does this.

File: src/App.tsx [hunk 3]
```diff
@@ -1,7 +1,27 @@
...
+import Container from 'react-bootstrap/Container';
+import 'bootstrap/dist/css/bootstrap.min.css';
 
-function App() {
-	return <div className="App">Hello Businext!</div>;
-}
+const App = () => {
+	return (
+		<div className="app">
+			<BrowserRouter basename={process.env.PUBLIC_URL}>
```
Comment (DawYeong) [inline src/App.tsx:13]: Is there a proper check for this? If it doesn't exist would it just display undefined?

File: src/index.tsx [hunk 4]
```diff
@@ -1,14 +1,14 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import './index.css';
+import "./styles/global.css";
```
Comment (DawYeong) [inline src/index.tsx]: Any reason as to why this is a double quotation?

File: src/App.tsx (hunk 3 above)
Comment (xujustinj) [reply src/App.tsx:13]: It always exists, according to the create-react-app gods who said so in [a comment in the template index.html](https://github.com/businext/businext-portal/pull/1/files?diff=unified&w=1#diff-c2cc24bc9001b11b6add48a4cd8f893d5d6c6e4d1bd254158bd14ab997f552cdR16-R25)

File: src/index.tsx (hunk 4 above)
Comment (xujustinj) [reply src/index.tsx]: Prettier has been spazzing out on me...

**Review by JoshuaC15 — DISMISSED** [review]

**Review by DawYeong — DISMISSED** [review]

File: src/components/BusinessForm.tsx [hunk 5]
```diff
@@ -0,0 +1,62 @@
+import { useState, ChangeEvent } from 'react';
+import Button from 'react-bootstrap/Button';
+import Form from 'react-bootstrap/Form';
+import { BusinessQuery } from '../models/Business';
+
+interface BusinessFormProps {
```
Comment (l2lam) [inline src/components/BusinessForm.tsx:6]: A common naming convention for interfaces is to prefix the name with a capital i 
```suggestion
interface IBusinessFormProps {
```

Comment (xujustinj) [reply src/components/BusinessForm.tsx:6]: Apparently TypeScript is one of the languages where this convention doesn't apply https://github.com/basarat/typescript-book/blob/master/docs/styleguide/styleguide.md#interface

**Review by ScottMorris — APPROVED** [review]

**Review by JoshuaC15 — APPROVED** [review]

**Review by DawYeong — APPROVED** [review]

**Review by tim-zwart — APPROVED** [review]

File: src/components/BusinessForm.tsx (hunk 5 above)
Comment (l2lam) [reply src/components/BusinessForm.tsx:6]: That's a terrible reason... vs at-a-glance association which is why we also do the capitalization conventions

**Pull request merged**